Player/GrenadeLauncher.gd

```diff
@@ -72,78 +71,73 @@ func update_aim() -> void:
 	var motion_down := to_target.y - peak_height
 	var time_going_down := sqrt(-2.0 * motion_down / gravity)
 	
-	var time_to_land := time_going_up + time_going_down
+	_time_to_land = time_going_up + time_going_down
 
 	var target_position_xz_plane := Vector3(to_target.x, 0.0, to_target.z)
 	var start_position_xz_plane := Vector3(_launch_point.position.x, 0.0, _launch_point.position.z)
 
-	var forward_velocity := (target_position_xz_plane - start_position_xz_plane) / time_to_land
+	var forward_velocity := (target_position_xz_plane - start_position_xz_plane) / _time_to_land
 	var velocity_up := sqrt(2.0 * gravity * motion_up)
 	
 	# Caching the found initial_velocity vector so we can use it on the throw() function
 	_throw_velocity = Vector3.UP * velocity_up + forward_velocity
 
-	# Redraw the grenade's motion preview
-#	_grenade_path.curve.clear_points()
-	_trail_points.clear()
+
+func _draw_throw_path() -> void:
 	const TIME_STEP := 0.05
-	var time_current := 0.0
-	var end_time := time_to_land + 0.5
-	while time_current < end_time:
-		var point := _throw_velocity * time_current + Vector3.DOWN * gravity * 0.5 * time_current * time_current
-		_trail_points.append(point)
-		time_current += TIME_STEP
-	_generate_mesh()
+	const TRAIL_WIDTH := 0.25
```

Comment:
I prefer us to use local constants when possible to avoid having to jump around to read the code